docs/table/construct_table.rst

```diff
@@ -691,9 +689,9 @@ meta
 
 The ``meta`` argument is an object that contains metadata associated
 with the table. It is recommended that this object be a dict or
-OrderedDict_, but the only firm requirement is that it can be copied with
+|OrderedDict|, but the only firm requirement is that it can be copied with
```

Comment:
So, here and just below would be the other places to replace with :class:`~collections.OrderedDict`

Reply:
✅